misc/prime_seive.c

```diff
@@ -0,0 +1,71 @@
+/**
+ * @file
+ * @brief [Prime Seive](https://leetcode.com/problems/count-primes/)
+ * algorithm implementation.
+ * @author[Divyansh Kushwaha](https://www.github.com/webdesignbydivyansh)
+ */
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+const int MAX_SIZE=1000000;
```

Comment:
```suggestion
constexpr int MAX_SIZE=1000000;
```

Reply:
i think it's better to write 1000000 rather than constexpr int MAX_SIZE because some tests are failing as you can see.
Please look into this matter

Reply:
yes please revert the changes;  i thought it is `cpp` repo